examples/nidaqmx/analog-input-every-n-samples-aio.py

```diff
@@ -101,15 +101,21 @@ def check_for_warning(response):
                 )
             )
 
-            # Wait for initial_metadata to ensure that the callback is registered before starting
-            # the task.
+            # Wait for initial_metadata and check for stream errors to ensure that the callback is
+            # registered before starting the task.
             await every_n_samples_stream.initial_metadata()
+            if every_n_samples_stream.done():
+                every_n_samples_response = await every_n_samples_stream.read()
+                check_for_warning(every_n_samples_response)
```

Comment:
I added `check_for_warning` for consistency with other code that gets responses, but on second thought it probably doesn't make sense here.